postgres/datadog_checks/postgres/statements.py

```diff
@@ -217,6 +245,9 @@ def _load_pg_stat_statements(self):
                 )
                 return []
 
+            if not self._track_io_timing_enabled():
```

Comment:
Can we update the integration tests to test both on & off and confirm that the metrics appear and disappear accordingly?

Reply:
Added these tests.  The `track_io_timing` value doesn't seem to be configurable at runtime so the tests modify `check.pg_settings` directly to trick the check into thinking that the value is what the test expects it to be.